redis/tests/test_cluster.rs

```diff
@@ -887,6 +887,37 @@ fn test_cluster_route_correctly_on_packed_transaction_with_single_node_requests2
     assert_eq!(result, expected_result);
 }
 
+#[test]
+fn test_cluster_with_client_name() {
+    let cluster = TestClusterContext::new_with_cluster_client_builder(
+        3,
+        0,
+        |builder| builder.client_name(RedisCluster::client_name().to_string()),
+        false,
+    );
+    let mut con = cluster.connection();
+    let client_info: String = redis::cmd("CLIENT").arg("INFO").query(&mut con).unwrap();
```

Comment:
i'm not sure if client info is being sent to all nodes, have you checked it?

Reply:
I think routing it to all nodes would cover the functionality of this feature in cluster better

Reply:
you can't route manually in sync cluster.

Reply:
We think that randomly checking a single node is good enough